metaflow/metaflow_config.py

```diff
@@ -347,6 +347,8 @@
 
 KUBERNETES_SANDBOX_INIT_SCRIPT = from_conf("KUBERNETES_SANDBOX_INIT_SCRIPT")
 
+OTEL_ENDPOINT = from_conf("OTEL_ENDPOINT")
```

Comment:
Setting this to an invalid OTEL URL will lead to a lot of the following error
`Failed to export batch code: 404, reason: 404 page not found`

and if the URL is completely unreachable, full stack traces from `requests` will be flooding the logs.

this is sort of expected behavior, but the errors could be cleaned up a bit.